.eslintrc.js

```diff
@@ -2,23 +2,21 @@ module.exports = {
   env: {
     browser: true
   },
-  extends: "eslint:recommended",
+  extends: ["plugin:prettier/recommended"],
+  plugins: ["prettier"],
   parserOptions: {
-    ecmaVersion: 2018,
-    sourceType: "module"
+    ecmaFeatures: {
+      jsx: true,
+      modules: true
+    }
   },
+  parser: "babel-eslint",
```

Comment:
This was needed to enable us to parse certain es6 syntax we use in some of our off-chain infrastructure like the bots.

Reply:
Was this the reason that changes in the bots were throwing several errors even after we ran `npm run lint`?